### ISSUE TITLE
Unable to locate the "Google" option in Preferences for IDE setup for App Engine

### ISSUE THREAD

**Issue by nkshschdv** [TEAMMATES/teammates#7307]
- **Operating System: Ubuntu/Debian 16.04**
- **JDK Version: 1.7 **
- **Eclipse / IntelliJ Version (if applicable): Neon.3 Release (4.6.3)**
- **Firefox / Chrome Version (where applicable):Chrome**

**What I was trying to do / What I need help with**
Problem in following guidlines at https://github.com/TEAMMATES/teammates/blob/master/docs/ide-usage.md .

I have a problem in Step 2 as I could not find the "Google " option for app engine in the preferences to proceed into further step. 
As seen in the following [screenshot]

![new](https://cloud.githubusercontent.com/assets/15321028/26069856/0edffe6c-39c0-11e7-86bf-22b1b01bbb63.png)

**Attempts to resolve the problem**
At first ,I thought it could be because of Linux distribution but in later steps i encountered another procedure to deal with the same option. I tried to find the issue and google search which Installed couple of plugins but of no use.

I had installed the following version of Eclipse on my system ,I have gut feeling that i had installed a wrong one. so , please check it out.
IDE Name and version:
Eclipse IDE for Java Developers

Version: Neon.3 Release (4.6.3)
Build id: 20170314-1500

Thank you !

**Comment by VamsiSangam** [comment]
Hi @nkshschdv ! Did you download the [Google plugin for Eclipse](https://developers.google.com/eclipse/docs/download) ?

**Comment by nkshschdv** [comment]
Actually , I found out that ,I had downloaded the wrong variant of Eclipse.
In Documents , its written "Eclipse IDE for **Java EE** developers version of Luna, Mars and Leon".
But I've installed "Eclipse IDE for Java Developers".So, I've corrected that.

Thanks , I have another problem , regarding i'm not able to add the "node_modules" folder in the Exclude Group in settings, Even though I've mentioned the folder in the validation settings.
